src/Config.php

```diff
@@ -8,18 +8,23 @@
 
 class Config extends BaseConfig
 {
-    public function getToken() : string
+    public function getToken(): string
     {
-        return $this->getValue(['parameters', '#token']);
+        return (string) $this->getValue(['parameters', '#token']);
     }
 
-    public function getUrl() : string
+    public function getUrl(): string
     {
-        return $this->getValue(['parameters', 'url']);
+        return (string) $this->getValue(['parameters', 'url']);
     }
 
-    public function isIncremental() : bool
+    public function isIncremental(): bool
     {
-        return $this->getValue(['parameters', 'incremental']);
+        return (bool) $this->getValue(['parameters', 'incremental']);
+    }
+
+    public function isFullSync(): bool
+    {
+        return (bool) $this->getValue(['parameters', 'fullSync']);
```

Comment:
s tím `fullSync` si nejsem jistej, není na první pohled jasný, co to znamená. není to vůbec popsaný v dokumentaci (update README chybí úplně) a pak i ten název mi přijde matoucí. takhle od oka bych to viděl v pojmenování na `update` (fullSync=false)/`replace` (fullSync=true).